packages/app-store/sendgridothercalendar/_metadata.ts

```diff
@@ -0,0 +1,26 @@
+import type { AppMeta } from "@calcom/types/App";
```

Comment:
@leog Can you try using app-store -cli to create the app. You can refer to https://developer.cal.com/guides/how-to-build-an-app. It would not add much code that's not required.
e.g. metadata is supposed to be pretty lean now.


Would love to fix any issues that come up with app-store-cli usage.

Reply:
Oh sure, I can create a new app using the CLI and then port what I have to it, so I leverage the latest structure you put together. It was not that I'm too lazy, as Sendgrid behaves the exact same way as closecom app, I copied the code from it 😅